src/server/test/web/readingsBarGroupQuantity.js

```diff
@@ -48,8 +48,24 @@ mocha.describe('readings API', () => {
                             graphicUnitId: unitId });
                     // Check that the API reading is equal to what it is expected to equal
                     expectReadingToEqualExpected(res, expected, GROUP_ID);
-            });
+                });
                 // Add BG4 here
+                mocha.it('13 day bars for 15 + 20 minute reading intervals and quantity units with +-inf start/end time & kWh as kWh', async () =>{
```

Comment:
Can you please add BG4: at the start of the mocha description so it is 'BG4: 13 day.......'.